packages/neos-ui/src/Containers/Modals/SelectNodeType/index.js

```diff
@@ -165,6 +165,17 @@ export default class SelectNodeType extends PureComponent {
 
     handleNodeTypeFilterChange = filterSearchTerm => this.setState({filterSearchTerm});
 
+    componentDidUpdate() {
```

Comment:
Why `componentDidUpdate` and not `componentDidMount`?

Reply:
On `componentDidMount` I was missing `this.state.insertMode` and/or the getAllowedNodeTypesByCurrentInsertMode result, I think.

Reply:
Ah, makes sense